Carbon/__tests__/components/EmissionModal.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import EmissionModal from '../../navigation/screens/Progress/EmissionModal';
+jest.mock('react-native/Libraries/Components/ToastAndroid/ToastAndroid', () => ({
+    show: jest.fn(),
```

Comment:
My main comment for these tests would be to add a some comments to better understand what is going on here.